shared/modules/transaction.utils.ts

```diff
@@ -245,7 +234,7 @@ export async function determineTransactionAssetType(
     try {
       // We don't need a balance check, so the second parameter to
       // getTokenStandardAndDetails is omitted.
-      const details = await getTokenStandardAndDetails(txMeta.txParams.to);
+      const details = getTokenStandardAndDetails(txMeta.txParams.to);
```

Comment:
Why the removal of the await? 🤔

Reply:
Looks like `getTokenStandardAndDetails` doesn't return a promise type.